joblib/parallel.py

```diff
@@ -678,7 +675,7 @@ def retrieve(self):
                     self._output.extend(job.get(timeout=self.timeout))
                 else:
                     self._output.extend(job.get())
-            except tuple(self.exceptions) as exception:
+            except Exception as exception:
```

Comment:
I am not sure I understand the implications of this change. Is it just trying to simplify the previous code?